frontend/src/components/layouts/Footer.vue

```diff
@@ -43,11 +43,9 @@
                                     <a @click="goTo('about_us')" class="cursor-pointer">About Us</a>
                                 </li>
                                 <li>
-                                    <a class="cursor-pointer" data-modal-toggle="contact_modal" data-modal-target="contact_modal">
+                                    <a @click="showContactModel = true" class="cursor-pointer">
                                         Contact
                                     </a>
-                                    <!-- Contact modal component -->
-                                    <ContactModel></ContactModel>
                                 </li>
                                 <li>
                                     <a href="#" class="cursor-pointer">Blue Mom</a>
```

Comment:
Based on customer feedback, this should lead to the view about_us and lead the user to the Blue Mom section,
Make sure the same link in the Header Component does the same

Reply:
Done!